images/linux/scripts/installers/haskell.sh

```diff
@@ -9,6 +9,7 @@ source $HELPER_SCRIPTS/etc-environment.sh
 
 # Any nonzero value for noninteractive installation
 export BOOTSTRAP_HASKELL_NONINTERACTIVE=1
+export BOOTSTRAP_HASKELL_INSTALL_NO_STACK_HOOK=1
```

Comment:
We may also want to check that `~/.stack/hooks/ghc-install.sh` does NOT exist. I'm not sure what's the best way in the test suite to do that.

Reply:
We should update Haskell tests and add this check - `"$HOME/.stack/hooks/ghc-install.sh" | Should -Not -Exist`
1. Ubuntu - https://github.com/actions/runner-images/blob/main/images/linux/scripts/tests/Haskell.Tests.ps1
2. macOS - https://github.com/actions/runner-images/blob/main/images/macos/tests/Haskell.Tests.ps1